doc/menu.md

```diff
@@ -0,0 +1,204 @@
+<!---
```

Comment:
I don't think this menu file belong here, asn it doesn't seem to serve any purpose. If the links actually worked, I guess that would make sense, but none of them do.